src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,397 @@
+period. This is the time between when the money is captured from the
+customer's credit card and when it is transferred to the provider's bank
+account. The payout in marketplaces usually happens after the provider
+has successfully provided the agreed service.
+
+Customer refund usually happens during this delayed payment period.
+There are many reasons why a refund may be necessary. For example, the
+provider or customer may not be able to make it, or the provided service
+was not what was agreed.
+
+### Step 5: Provider payout
+
+If everything in the transaction went right and the customer received
+the agreed service, the money from the payment gateway will be
+eventually paid out to the provider.
+
+## Stripe default integration
+
+In the default Flex implementation, the process described above is
+implemented using Stripe. The integration uses
+[Stripe Custom Connect accounts](https://stripe.com/docs/connect/custom-accounts)
+as the provider’s payout account. The customer can checkout using a
+payment card, and they can also save their payment method for future
+use. The integration uses
+[Stripe destination charges](https://stripe.com/docs/connect/destination-charges)
+to collect the payment to the platform’s account first. Once the
+transaction is successfully over, the provider’s share of the payment is
+paid through the provider’s Custom Connect account to the bank account
+they provided upon onboarding.
+
+### Default payment process with Stripe
+
+![Default payment flow in Flex](./automatic_confirmation_flow.png)
+
+#### 1. Provider onboarding
+
+In the Flex default integration, users need to have a Stripe account
+with a bank account set up before others can initiate transactions with
+them successfully. This is done by
+[creating a Stripe account](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account)
+for the authenticated user. The
+[FTW templates](http://localhost:8000/cookbook-payments/provider-onboarding-and-identity-verification/)
+are configured to do this step out-of-the-box using
+[Stripe Connect Onboarding](https://stripe.com/en-fi/connect/onboarding).
+
+To create the account, Stripe requires verification information from the
+provider, and the specific types of verification information depends on
+the provider's country. You can check the verification requirements for
+your most likely marketplace provider demographics in
+[Stripe's own documentation](https://stripe.com/docs/connect/required-verification-information).
+
+#### 2. Customer checkout
+
+When the customer initiates a transaction in the Flex default
+transaction processes, Flex creates a
+[PaymentIntent](/background/payment-intents/) for the total price of the
+transaction. Once the PaymentIntent is confirmed, Stripe preauthorizes
+the sum from the customer's payment method. In other words, the sum does
+not leave the customer's account, but it is still reserved and not
```

Comment:
s/account/credit card

Or can money leave credit card? Would it be more along the lines of "the sum is not paid from the customer's credit card, but it still..."

Reply:
Wording fixed